packages/markdown-docx/test/data/ciceroMark/ciceroMark.js

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
This is just test data. You don't need a license here.

Reply:
Convert this file so that it only contains JSON data. Using JS is just making it verbose.

Reply:
Yeah, that would be better. Separate files for each test.